FontConfig.go

```diff
@@ -89,3 +89,12 @@ func (config FontConfig) SetMergeMode(value bool) {
 		C.iggFontConfigSetMergeMode(config.handle(), castBool(value))
 	}
 }
+
+// getFontDataOwnedByAtlas gets the current ownership status of the font data.
+func (config FontConfig) getFontDataOwnedByAtlas() bool {
+	if config != DefaultFontConfig {
+		return C.iggFontConfigGetFontDataOwnedByAtlas(config.handle()) != 0
+	}
+
+	return true
```

Comment:
For future reference:
The "quick" path would have been the inverse:
```
if config == DefaultFontConfig {
  return true
}
return C.iggFontConfig...
```
Because in such a case the "simple" case is first, which even the compiler could optimize. (I believe it was 1.13 that added compiler optimization which inlines guard clauses)
I'll accept the pull request and invert myself.

Reply:
oh - sorry, I was wrong.
This variant is optimized as well - and is also in style of all the other implementations.
I'll keep it.